source/dhtslib/gff/package.d

```diff
@@ -0,0 +1,176 @@
+/**
+
+GFF3 Record abstraction
+
+This module provides a readable, writeable abstraction of GFF3 records.
+
+For reading and writing from and to bgzip-compressed/tabix-indexed files,
```

Comment:
looks like straight pull from gff3d? need to remove

Reply:
Do you mean remove all gff3d code or just the descriptive text? I would have just updated gff3d and included it as a dependency. The problem is gff3d would need access to dhtslib.coordinates and that creates a cyclic dependency that dub can't solve.